### PR TITLE
Added docs Prerequisites section

### PR DESCRIPTION
**Pull request by SergeyKleyman**

### CODE REVIEW

File: docs/setup.asciidoc [hunk 1]
```diff
@@ -6,13 +6,31 @@ endif::[]
...
 == Set up the Agent
 
-Install the agent using one of the https://github.com/elastic/apm-agent-php/releases/latest[packages for supported platforms].
+[discrete]
+[[setup-prerequisites]]
+=== Prerequisites
+
+[discrete]
+==== Operating system
+The agent is currently only available for Linux operating system.
```
Comment (bmorelli25) [inline docs/setup.asciidoc:15]:
```suggestion
The agent is currently only available for the Linux operating system.
```

File: docs/setup.asciidoc [hunk 2]
```diff
@@ -6,13 +6,31 @@ endif::[]
...
+
+[discrete]
+==== PHP
+The agent supports PHP versions 7.2-8.1
```
Comment (bmorelli25) [inline docs/setup.asciidoc:19]:
```suggestion
The agent supports PHP versions 7.2-8.1.
```

**Review by bmorelli25 — APPROVED** [review]
Thanks @SergeyKleyman.

**Comment by SergeyKleyman** [comment]
@bmorelli25 Thank you for the review and suggestions.

**Pull request merged**